tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -95,7 +124,7 @@ func checkEventType(ev map[string]interface{}) (interface{}, error) {
 func handler(ctx context.Context, ev map[string]interface{}) error {
 	event, err := checkEventType(ev)
 	if err != nil {
-		fmt.Println("invalid event: %s", ev)
+		fmt.Printf("invalid event: %s", ev)
```

Comment:
this should still be `fmt.Println`

Reply:
Will revert. For some reason the compiler is warning me that the Println call has possible formatting directive %s.

Reply:
oh yeah, right it should be `Printf` but we need `\n` at the end.

Reply:
updated here as well as in s3.go